wurst/data/BitSet.wurst

```diff
@@ -47,3 +62,28 @@ public function bitset.remove(int v) returns bitset
 
 @test function testRemove()
 	bitset(45).remove(3).val.assertEquals(37)
+
+@test function testFlip()
+	emptyBitset().add(3).flip(3).contains(3).assertFalse()
+	emptyBitset().flip(3).contains(3).assertTrue()
```

Comment:
Maybe add some tests for boundary conditions

Reply:
I've added a test.